public/categories/index.html

```diff
@@ -0,0 +1,102 @@
+<!doctype html>
```

Comment:
AppEngine just tars up an entire directory and deploys that. There is no container involved like before so there's nothing in the pipeline that would actually run a hugo process. 

One option would be to have Cloud Build run a `hugo` container as a build step and then save all the generated public content to a tmp workspace before the `gcloud app deploy` step, but it would be more of a mystery what public files would actually be built and submitted.